index.html

```diff
@@ -0,0 +1,172 @@
+                      </article>
+                    </div>
+                  </div>
+                </div>
+            </div>
+        </section>
+        <section id="about-section">
+            <div class="container-fluid about-page p-0">
+                <div class="container-md">
+                  <div class="row">
+                    <div class="col-12 p-0 m-0">
+                      <article class="mt-3 pt-5 about-section">
+                        <h2 class="mt-4 py-2">SOFTWARE DEVELOPMENT ONLINE BOOTCAMP</h2>
+                        <div class="my-4 about-line"></div>
+                      </article>
+                      <div class="about-cards mt-5">
+                        <div class="about-item">
+                          <a href="#"><img src="imgs/html.png" alt="about-image1" height="84"></a>
+                          <div class="d-flex flex-column">
+                            <h3>HTML</h3>
+                          <p> Fast paced get to the point training, to learn about creating websites and using HTML code. By the end of the course you will have the skills and know how to apply HTML to make a real website.</p>
+                          </div>
+                        </div>
+                        <div class="about-item">
+                          <a href="#"><img src="imgS/CSS3.svg" alt="about-image1" height="84"></a>
+                          <div class="d-flex flex-column">
+                            <h3>CSS</h3>
+                          <p>CSS, or Cascading Style Sheet, is responsible for the styling and looks of a website. In this course, you will learn CSS from the ground up</p>
+                          </div>
+                        </div>
+                        <div class="about-item">
+                          <a href="#"><img src="imgs/js.png" alt="about-image1" height="84"></a>
+                          <div class="d-flex flex-column">
+                            <h3>JavaScript</h3>
+                          <p>JavaScript is an object-oriented programming language employed by most websites along with HTML and CSS to create robust, dynamic and interactive user experiences.</p>
+                          </div>
+                        </div>
+                        <div class="about-item">
+                          <a href="#"><img src="imgs/algorithm.jpg" alt="about-image1" height="84"></a>
+                          <div class="d-flex flex-column">
+                            <h3>Algorithm</h3>
+                          <p>Learn how to structure and use algorithms to solve real life problems.</p>
+                          </div>
+                        </div>
+                      </div>
+                      <a href="#" class="d-block text-white text-center my-5 fs-5">SEE THE COMPLETE COURSES</a>
+                    </div>
+                  </div>
+                </div>
+            </div>
+        </section>
+        <section id="about-section1">
```

Comment:
- [ ] The featured speakers section, which should be rendered using javascript is completely missing from this project, please add it to comply with the requirements:

![image](https://user-images.githubusercontent.com/38522578/137812632-7d8bbc81-faff-4304-949a-e93aafc28b6e.png)

![image](https://user-images.githubusercontent.com/38522578/137813337-0742d0f8-6643-427e-8cf2-5c8d623127ae.png)